packages/components/src/index.ts

```diff
@@ -2,6 +2,7 @@ export { default as Banner } from "./Banner";
 export { default as Button } from "./Button";
 export { default as Clickable } from "./Clickable";
 export { default as Heading } from "./Heading";
+export { default as SvgIcon, IconProps } from "./SvgIcon";
```

Comment:
Random thought - if we get enough icons, one day we could think about creating a logical separation between icons and eds components, and create a `@chanzuckerberg/eds-icons` package.

Reply:
💡 !